docs/index.md

```diff
@@ -0,0 +1,43 @@
+# What is MCAP?
```

Comment:
I don't think the main homepage at `/index.html` needs to start out with a question, it can just explain what MCAP is. Either:
```suggestion
# MCAP
```

or:
```suggestion
```